code/WorkInProgress/MarqStuff.dm

```diff
@@ -708,10 +676,10 @@
 		loaded.set_loc(A)
 		current_projectile.implanted = A
 		current_projectile.material = copyMaterial(loaded.head_material)
-		var/default_power = 12
+		var/default_power = 20
 		if(loaded.head_material)
 			if(loaded.head_material.hasProperty("hard"))
-				current_projectile.power = round(loaded.head_material.getProperty("hard") / 2.6) //40hard ~ 15dmg, 80hard ~ 30dmg
+				current_projectile.power = round(20+loaded.head_material.getProperty("hard") / 4) //40hard ~ 15dmg, 80hard ~ 30dmg note this has now been modified by anacroniser for it to be 30 and 40 respectively
```

Comment:
Not sure how I feel about squishifying the effect of material stats here.

Reply:
I think thats a fair enough point but from monkey testing it made mauxite usable but notably inferior to say, uqill. My goal in that choice was to have consistent usable behavior. Perhaps change the flat 20 to 10 and increase scaling by 50%. I'm not sure as to what is right but I don't feel like the lower end of the spectrum (e.g. mauxite and steel) is particularly powerful even with that flat 20. I'd love extra input.

Reply:
Also as a bit of an aside, I think having this compressed value encourages usage of the refinery by non-miners, hopefully if this continues as a trend that will lead to miners actually interacting with the rest of the station. Just something that came to mind.

Reply:
Perhaps even just keep the 20 flat and bring the scaling back to `hard/2.6`. Also, might be nice if the material stats of the shaft mattered for something (density affecting dissipation delay and projectile speed by +- 20% or so?) idk.

Reply:
As of the current the shaft material does transfer material properties like the teleportation of telecrystal or the explosive nature of erebite. I think having a free slot to build in radioactivity or other debuffing effects is interesting and presents unique functionality to bows. I will test the scaling back at 2.6, it may feel a bit too powerful with something like uqill however.

Reply:
From testing, /2.6 actually feels relatively balanced so I'll leave it at that.